models/classes/class.QtiTestService.php

```diff
@@ -338,7 +339,7 @@ private function generateIdentifier(XmlDocument $doc, $qtiType, $offset = 1)
      * @throws common_exception_Error
      * @throws common_exception_FileSystemError
      */
-    public function importMultipleTests(core_kernel_classes_Class $targetClass, $file)
+    public function importMultipleTests(core_kernel_classes_Class $targetClass, $file, bool $overwriteTest = false, ?string $itemClassUri = null)
```

Comment:
Aren't we adding ad-hoc parameters in methods just because some specific need appeared? Is it sustainable? 

We should pass a **command** or query **parameter** object instead with the required/optional data. 

Check this https://dev.to/thinkster/code-smell-too-many-parameters-435e

Alternatively you can add setters to the the service to consider this cases.